### PR TITLE
feat: template create automation

### PR DESCRIPTION
**Pull request by kumiega**
I have created a more advanced template automation inside create-template.js script. I added two extra script paths with parameters usage. The `all` parameter will create tasks for whole event and `with-previous` do it from relative "today" to the event start date.

Also I have prepared the npm scripts for these parameters and updated the `README.md` docs.

### CODE REVIEW

File: scripts/create-template.js [hunk 1]
```diff
@@ -1,27 +1,26 @@
...
 
-const folderPath = path.join('tasks', folderName);
+if (!fs.existsSync("tasks")) {
+  fs.mkdirSync("tasks");
+}
 
-if (!fs.existsSync('tasks')) {
-    fs.mkdirSync('tasks');
+if (!arg) {
+  createTaskFolder(new Date());
```
Comment (przeprogramowani) [inline scripts/create-template.js:11]: As a follow-up, would be nice to reuse `createTaskFoldersFromRange` with just one day. I do understand it requires more work, so only FYI, not a blocker.

File: README.md [hunk 2]
```diff
@@ -14,14 +14,22 @@ Zadania są dostępne na stronie [opanujfrontend.pl/advent](https://opanujfronte
 
 Każdego dnia wykonaj w repozytorium polecenie `npm run create` a następnie skopiuj kod ze strony.
 
+Użyj komendy `npm run create all`, jeśli chcesz wygenerować foldery z plikami startowymi
```
Comment (przeprogramowani) [inline README.md:17]: I think it's best to reference explicit names of npm scripts just to be consistent. 

`npm run create:month` as suggested by @MarcinParda
`npm run create:?` - I can't find any single-word name for "up until today"

**Comment by przeprogramowani** [comment]
@kumiega Thank's a lot for this PR. Just to preserve original order of requests and proposals, I just merged `create:month` from @MarcinParda - I think you PR is a natural extension of what's already in master so if you don't mind, please continue and let's make the "with previous" happen. Happy to merge once you review my comments.

**Comment by kumiega** [comment]
Thanks for response. I will try to make changes today.

**Comment by kumiega** [comment]
It took some time, but I finally pushed the changes to the repo!

**Comment by psmyrdek** [comment]
We'll take a look today, many thanks :)

**Comment by przeprogramowani** [comment]
@all-contributors please add @kumiega for code

**Pull request merged**